src/app/functions/server/getDataBrokerRemovalTimeEstimates.ts

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { OnerepScanResultRow } from "knex/types/tables";
+import { LatestOnerepScanData } from "../../../db/tables/onerep_scans";
+
+interface DataBrokerRemovalTime {
+  d: OnerepScanResultRow["data_broker"];
+  t: number;
+}
+
+export function getDataBrokerRemovalTimeEstimates(
+  scanData: LatestOnerepScanData,
+) {
+  const removalTimeData = JSON.parse(
+    process.env.DATA_BROKER_REMOVAL_ESTIMATES_DATA as string,
+  ) as DataBrokerRemovalTime[];
```

Comment:
I wonder if it might be a good idea to wrap this into a `try ... catch` and just return an empty array if it fails (with a `logger.error` log), just to prevent syntax errors when modifying the env var from crashing the entire app?

Reply:
Yes, let’s add this: [60bc27a](https://github.com/mozilla/blurts-server/pull/5189/commits/60bc27a2f861934540845afc124604f38bbcbc6c).